src/com/t_oster/liblasercut/drivers/FullSpectrumCutterProperty.java

```diff
@@ -0,0 +1,201 @@
+/**
+ * This file is part of LibLaserCut.
+ * Copyright (C) 2011 - 2014 Thomas Oster <mail@thomas-oster.de>
+ *
+ * LibLaserCut is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * LibLaserCut is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with LibLaserCut. If not, see <http://www.gnu.org/licenses/>.
+ *
+ **/
+package com.t_oster.liblasercut.drivers;
+
+import com.t_oster.liblasercut.LaserProperty;
+
+/**
+ * Handles parameters for full spectrum jobs.
+ * 
+ * Based upon PowerSpeedFocusProperty by Thomas Oster <thomas.oster@rwth-aachen.de>
+ * 
+ * @author Volkan Vonk <vol.vonk@yandex.com> 
+ */
+public class FullSpectrumCutterProperty implements LaserProperty
+{
```

Comment:
This class could be used on other devices and should thus be named `FloatPowerSpeedProperty` in order to be consistent with the other properties.